### ISSUE TITLE
Do you get the same accuracy as original paper?

### ISSUE THREAD

**Issue by TheSunWillRise** [sinAshish/LGM-Net-Pytorch#1]

**Comment by sinAshish** [comment]
I haven't tried training the model yet.

**Comment by EternalWang** [comment]
Have you tried training the model?
Thanks!

**Comment by sinAshish** [comment]
when tested with FC100 dataset, it performs almost similar results as the miniImageNet results mentioned in the paper

**Comment by EternalWang** [comment]
Thanks for you reply.
But CIFAR-100 is different from miniImageNet, so I think it isn't suitable to compare results on CIFAR-100 to miniImageNet even the auther didn't test on the former.
I'll help to train the model on miniImageNet if I have time.

**Comment by lailvlong** [comment]
> Thanks for you reply.
> But CIFAR-100 is different from miniImageNet, so I think it isn't suitable to compare results on CIFAR-100 to miniImageNet even the auther didn't test on the former.
> I'll help to train the model on miniImageNet if I have time.

Could you please share the results and codes when you finish it?

**Comment by EternalWang** [comment]
> > Thanks for you reply.
> > But CIFAR-100 is different from miniImageNet, so I think it isn't suitable to compare results on CIFAR-100 to miniImageNet even the auther didn't test on the former.
> > I'll help to train the model on miniImageNet if I have time.
> 
> Could you please share the results and codes when you finish it?

Sorry, I'm a little busy these days. Maybe I'lI finish it a month later.

**Comment by wxidij** [comment]
> > > Thanks for you reply.
> > > But CIFAR-100 is different from miniImageNet, so I think it isn't suitable to compare results on CIFAR-100 to miniImageNet even the auther didn't test on the former.
> > > I'll help to train the model on miniImageNet if I have time.
> > Could you please share the results and codes when you finish it?
